test/testutils/syncenginetestutils.h

```diff
@@ -578,7 +585,14 @@ class FakeFolder
         return execUntilFinished();
     }
 
+    bool applyLocalModifications() Q_REQUIRED_RESULT
```

Comment:
We use c++17 on master so I guess we should start using https://en.cppreference.com/w/cpp/language/attributes/nodiscard